packages/common/src/polyfills/version/release-version.ts

```diff
@@ -0,0 +1,39 @@
+// default return values
+const OLD_DEV = [-1 ,0 ,0];
+const DEV = [0, 0, 0]
+
+/**
+ * get the console release version
+ * 
+ * @returns a tupple with the console release version, on none release it will return 0, 0, 0,
+ *          on old development server, return -1, 0, 0
+ */
+const getReleaseVersion = () => {
+    const releaseVersion = window['SERVER_FLAGS']?.releaseVersion;
+    const consoleVersion = window['SERVER_FLAGS']?.consoleVersion;
+
+    if (!releaseVersion || typeof releaseVersion !== 'string' || releaseVersion === '') {
```

Comment:
1.  empty string is already falsy
2.  `v6.0.6-20` looks like a magic variable:) please comment about origin/reason
3. the method can be unit-tested if we provide input as params i.e. `getReleaseVersion(releaseVersion,consoleVersion)`

Reply:
> empty string is already falsy

I added it because I like to be extra explicit, but you are right it's unnecessary, i'll remove it

> v6.0.6-20 looks like a magic 

it is magic :-) . i'll add a comment :-)

> the method can be unit-tested

ahh, ok